handler/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/zeromicro/zero-contrib/handler
+
+go 1.18
```

Comment:
Please use go 1.16 for consistency.

Reply:
Thanks for the feedback, downgrade to 1.16